### ISSUE TITLE
-list-address option doesn't show the recipient/s list in the email reception

### ISSUE THREAD

**Issue by AndresLoGo** [muquit/mailsend#92]
The recipient/s gets de email but the list of recipient/s is empty in the message

**Comment by muquit** [comment]
It is working as designed. When ```-list``` option is used, mailsend works like a mailing list, it feeds each address  to SMTP server with ```RCPT TO:address```. If you want the addresses to show up, you have to write a wrapper script with comma separated addresses in ```-to``` or ```-cc``` option.
Thanks.

**Comment by AndresLoGo** [comment]
OK. But the reason because i try to use this way is because the options
that you say doesn't work properly as I refer in issue #93

2016-02-25 18:28 GMT+01:00 muquit <notifications@github.com>:

**Comment by muquit** [comment]
ok. I will look into it soon. Would you run mailsend in verbose mode (-v) while sending the mails? Do you see any error from the SMTP server?

Thanks.

**Comment by AndresLoGo** [comment]
I`ll try it when I can and send you the results.

Thank for all.

2016-02-26 15:47 GMT+01:00 muquit <notifications@github.com>:

**Comment by muquit** [comment]
I tested with smtp.gmail.with with 50 addresses in the file and did not see any issue. Also if you are running mailsend in Windows and if any address has 8 bit characters, it will not work. It's a known bug in Windows version of mailsend. 

Thanks.

**Comment by AndresLoGo** [comment]
Yes. In file the process works OK, in the other issue the test is without
file, where is the other issue I report.
The bug that you refer can be fixed?

2016-02-29 16:59 GMT+01:00 muquit <notifications@github.com>:
